dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/configurations/OpenAPIConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+package org.dataland.datalandqaservice.configurations
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeIn
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType
+import io.swagger.v3.oas.annotations.info.Info
+import io.swagger.v3.oas.annotations.security.OAuthFlow
+import io.swagger.v3.oas.annotations.security.OAuthFlows
+import io.swagger.v3.oas.annotations.security.SecurityScheme
+import io.swagger.v3.oas.annotations.servers.Server
+
```

Comment:
This file is also a candidate to be templated, and be generated for each new service, since it seems to me that is almost the same everytime.

Reply:
Should we create a stroy for this, or is it unnecessary or is the effort-reward-ratio not good enough. What do you think?

Reply:
It would be great to have a basic spring container with a database autogenerated via a gradle task or so.

Reply:
Ok do you want to create a story for this, or just present the idea in the next refinement?

I don't see this happening until the launch in October anyways.

As soon as you have created a story, or taken this as homework to present the idea in the next refinement (I will be on vacation), I will close this conversation.

Reply:
Florian hat sich notiert, diese Idee im nächsten refinement meeting in die Runde zu werfen.

Reply:
Discussion result: It would be nice to have. Won't be done because it would not pay off too much.